src/storage/txn/actions/mod.rs

```diff
@@ -6,10 +6,507 @@
 //! for archiving a certain target
 
 pub mod commit;
+pub mod pessimistic_prewrite;
 pub mod prewrite;
-pub mod shared;
+pub(crate) mod shared;
 
-#[cfg(test)]
 pub mod tests {
-    // Todo: move tests in mvcc/txn.rs which tests cooperation of different actions here
+    use super::*;
```

Comment:
I doubt whether it is a good idea to put all test functions here, this file will grow very large soon.